### PR TITLE
misc ui

### PR DESCRIPTION
**Pull request by mschneider**
* pull token labels in front of market ids
* ci for debug-ui (lint, format, build)
* fix imports in debug-ui to new client package layout
* install client dependencies before building during debug-ui preinstall
* workaround for eslint/commonjs import issue: https://github.com/eslint/eslint/discussions/17215

### CODE REVIEW

File: debug-ui/package.json [hunk 1]
```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "preinstall": "cd ../ && yarn build",
+    "preinstall": "cd ../ && yarn install --frozen-lockfile && yarn build",
     "dev": "next dev",
     "build": "next build",
+    "format": "prettier -c app/ lib/ scripts/",
+    "format:fix": "prettier -w app/ lib/ scripts/",
```
Comment (TovarishFin) [inline debug-ui/package.json:10]: is there a reason why you are not simply using the lint and lint:fix commands?

Comment (TovarishFin) [reply debug-ui/package.json:10]: ah it looks like prettier is not used in there... might as well combine the two though. or does this affect some CI stuff? I would prefer lint and lint:fix to handle formatting as well and just have the single set of commands.

**Review by TovarishFin — CHANGES_REQUESTED** [review]
looks good to me other than the format commands.

File: debug-ui/package.json (hunk 1 above)
Comment (mschneider) [reply debug-ui/package.json:10]: could combine into a higher level taks "check" & "fix". would prefer them separate for ci

**Review by TovarishFin — APPROVED** [review]

**Pull request merged**